.github/workflows/tests.yml

```
@@ -152,6 +154,109 @@ jobs:
           echo "Invalid command"
           exit 1
         fi
+          
+  cypress:
+    name: Cypress
+    runs-on: ubuntu-20.04
+    env:
+      MAILER_PASSWORD: ${{ secrets.MAILER_PASSWORD }}
+      MAILER_HOST: ${{ secrets.MAILER_HOST }}
+      MAILER_USER: ${{ secrets.MAILER_USER }}
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: mautictest
+        ports:
+          - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set DB port to env
+      run: echo "DB_PORT=${{ job.services.mysql.ports[3306] }}" >> $GITHUB_ENV
+
+    - name: Setup PHP, with composer and extensions
+      uses: shivammathur/setup-php@v2
+      with:
+        # Note: if you update the PHP version here, make sure to update libapache-mod-php7.4 below as well!
+        php-version: 7.4
+        extensions: mbstring, xml, ctype, iconv, intl, pdo_sqlite, mysql, pdo_mysql
```

@dennisameling I think that the configuration test is failing because it is timing out (it throws a server error at almost exactly 30 seconds) - is this the correct way to bump up the value in to php.ini file?

```suggestion
        extensions: mbstring, xml, ctype, iconv, intl, pdo_sqlite, mysql, pdo_mysql
        ini-values: max_execution_time=180
```

---

Let's try!